### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by poutnikl**
I added some enhancement to the Developer's guide, partly from my glossary.
Feel free to accept, modify or reject.

### CODE REVIEW

File: misc/readmes/profile_developers_guide.txt [hunk 1]
```diff
@@ -86,6 +93,14 @@ the routing engine:
    - turnInstructionCatchingRange  default 40m
    - turnInstructionRoundabouts     default=true=generate explicit roundabout hints
 
+  - variables to modify BRouter behaviour
+  
+   - processUnusedTags  ( default is false )
+         If an OSM tag is unused within the profile, BRouter totally ignores the tag existence
+         what increases BRouter speed. As a side effect, the tag is not even listed
```
Comment (Phyks) [inline misc/readmes/profile_developers_guide.txt]: "the tag existence which improves"

File: misc/readmes/profile_developers_guide.txt [hunk 2]
```diff
@@ -86,6 +93,14 @@ the routing engine:
...
+         in the route segment table nor the table exported as CSV. Setting it to true/1,
+         Brouter-web Data page listed all tags present in the RD5 file.
```
Comment (Phyks) [inline misc/readmes/profile_developers_guide.txt]: "Brouter-web Data page will list all tags present in the RD5 file."

File: misc/readmes/profile_developers_guide.txt [hunk 3]
```diff
@@ -199,10 +215,100 @@ The initial cost classifier
 
 To trigger the addition of the "initialcost", another variable is used:
 "initialclassifier" - any change in the value of that variable leads
-to adding the value of "initialcost".
+to adding the value of "initialcost". 
+
+An useful case may be an initial cost four mounting/dismounting a bicycle,
```
Comment (Phyks) [inline misc/readmes/profile_developers_guide.txt]: "A useful case may be an initial cost for mounting/dismounting"

File: misc/readmes/profile_developers_guide.txt [hunk 4]
```diff
@@ -199,10 +215,100 @@ The initial cost classifier
...
 For backward compatibility, if "initialclassifier"  = 0, it is replaced
 by the costfactor.
 
+The priority classifier
+-----------------------
+
+Priorityclassifier is a BRouter numerical parameter
+calculated for ways and used for generation of pictogram/voice navigation instructions. 
+
+Higher values means the more significant(noticable) way,
```
Comment (Phyks) [inline misc/readmes/profile_developers_guide.txt]: noticeable

File: misc/readmes/profile_developers_guide.txt [hunk 5]
```diff
@@ -230,6 +343,8 @@ or set up a local installation.
 BRouter-Web has a window at the lower left corner with a "Profile"
 and a "Data" tab. Here, you can upload profile scripts and see
 the individual cost calculations per way-section in the "Data"-tab.
+Activate eventually for the provile debugging "assign processUnusedTags = true"
```
Comment (Phyks) [inline misc/readmes/profile_developers_guide.txt]: for the profile

File: misc/readmes/profile_developers_guide.txt [hunk 6]
```diff
@@ -271,3 +386,9 @@ and that values are only appended at the end of the value lists.
 This is because the routing data files adress tags and values
 by their sequence numbers, so changing sequences would produce
 garbage data.
+
+Other resources
+---------------
+
+See https://github.com/poutnikl/Brouter-profiles/wiki/Glossary
+as a complementary inofficial source about various profile internals.
```
Comment (Phyks) [inline misc/readmes/profile_developers_guide.txt]: unofficial

File: misc/readmes/profile_developers_guide.txt (hunk 2 above)
Comment (poutnikl) [reply misc/readmes/profile_developers_guide.txt]: Suggestions applied and pushed.

**Pull request merged**